### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by parvez-rashid**

### CODE REVIEW

**Comment by mmanjuprasad** [comment]
better to add binary or png type of files
Keep a working show now button which takes you to products page
you can ignore this if u havent created one yet.

**Review by mmanjuprasad — APPROVED** [review]
better to add binary or png type of files
Keep a working show now button which takes you to products page
you can ignore this if u havent created one yet.

**Comment by parvez-rashid** [comment]
> better to add binary or png type of files Keep a working show now button which takes you to products page you can ignore this if u havent created one yet.

Yes I haven't linked the button yet, will do it. As for the images, I just included the images I could get, I'll try to replace non-png, non-binary images.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    <nav class="navigation">
+      <div class="nav-logo">SANGUINE store</div>
+      <label for="search" class="text-size-sm">
+        <input
+          type="search"
+          class="search-box padding-md border-rad-md"
+          name="search"
+          id="search"
+          placeholder="What are you looking for?"
+        />
```
Comment (Sahil25061999) [inline index.html:24]: use some border for the search container or some different background color, rest looks good

**Review by Sahil25061999 — APPROVED** [review]

**Pull request merged**